Composer/packages/client/src/components/Notifications/NotificationCard.tsx

```diff
@@ -148,27 +156,53 @@ export type NotificationProps = {
 };
 
 const makeLinkLabel = (link: NotificationLink) => (
-  <ActionButton css={linkButton} onClick={link.onClick}>
+  <ActionButton styles={linkButton} onClick={link.onClick}>
     {link.label}
   </ActionButton>
 );
 
 const defaultCardContentRenderer = (props: CardProps) => {
-  const { title, description, type, link, links } = props;
+  const { title, description, type, link, links, stretchLinks } = props;
+
+  const linkList = links ?? [link ?? null];
```

Comment:
Can you explain what is happening here? It looks like this is either going to resolve to `links` or  `[null]`. Should this be `links ?? [null]`?

Reply:
`links` is an array of links, while `link` is a single link; both are props on this component, meant for one or the other to be used. The weird `link ?? null` notation is for a type-checking thing (undefined vs null) - this would be more intuitively written as `links ?? [link]`. I think I can massage the types a bit better to make that work.